unittests/test_tags.py

```diff
@@ -212,6 +208,25 @@ def test_import_and_reimport_with_tags(self):
         for tag in tags:
             self.assertIn(tag, response["tags"])
 
+    def test_import_reimport_multipart_tags(self):
```

Comment:
The test only tests `import`, 
```suggestion
    def test_import_multipart_tags(self):
```

Reply:
But is this test really testing something different than: 

https://github.com/DefectDojo/django-DefectDojo/blob/05a74d5279e1d910b0f6bb4f6c1300a23f255e61/unittests/test_tags.py#L192-L213 ?

Reply:
Yep, it is testing tags with a comma in one of the tags where the tag came from a string rather than a list. This is a good indicator that the tags were supplied from a multipart form

Reply:
Ah yes.